torch/ao/quantization/fx/_model_report/detector.py

```diff
@@ -1079,11 +1085,13 @@ def _calculate_outlier_info(
             percentile_ratios (torch.Tensor): The average percentile_ratios per channel calculated by the observer
             counted_batches (torch.Tensor): The number of batches used for average calculation per tensor
             total_batches (int): The total number of batches that passed through observer in this epoch
+            constant_channels (torch.Tensor): The counts for constant batches per channel
 
         Returns a dictionary mapping:
             "outliers_detected" : list of bools per channel that are true if it is considered an outlier
             "sufficient_batches": if o_r was >= statistical_threshold:
                 where o_r = counted_batches / total_batches
+            "constant_channels": if non_constant_per_channel != total_batches for each channel
```

Comment:
What is `non_constant_per_channel`? I couldn't find any references to this anywhere

Reply:
Also is this a list of bools or a list of counts? Should clarify in comments